mods/doors/init.lua

```diff
@@ -49,19 +49,19 @@ function doors.get(pos)
 				if self:state() then
 					return false
 				end
-				return _doors.trapdoor_toggle(self.pos, player)
+				return _doors.trapdoor_toggle(self.pos, nil, player)
 			end,
 			close = function(self, player)
 				if not self:state() then
 					return false
 				end
-				return _doors.trapdoor_toggle(self.pos, player)
+				return _doors.trapdoor_toggle(self.pos, nil, player)
 			end,
 			toggle = function(self, player)
-				return _doors.trapdoor_toggle(self.pos, player)
+				return _doors.trapdoor_toggle(self.pos, nil, player)
 			end,
 			state = function(self)
-				return node_name:sub(-5) == "_open"
+				return minetest.get_node(self.pos).name:sub(-5) == "_open"
```

Comment:
I've essentially reverted this removal, in case the door-object would change its position at some point after creation. This forces a new lookup on self.pos. It's quite possible, that this is unnecessary, but i didn't want to risk it, and i don't have unit tests for _game api yet either. :-/